### ISSUE TITLE
`<details>` elements remain closed when exporting to PDF

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#6254]
## Environment

Joplin version: 2.7.13
Platform: Linux
OS specifics: PoP! OS 21.10/Flatpak

## Steps to reproduce

1. Create a new note
2. Add the following content:
```html
<details><summary>This should expand when exported</summary>
If it doesn't expand, we lack content in the exported PDF.
</details>
```
3. Export to PDF (`Ctrl+Shift+P` > `exportPdf`)
4. View the PDF

## Describe what you expected to happen

The dropdown should be expanded in the exported PDF.

## Logfile

Should not be applicable

**Comment by personalizedrefrigerator** [comment]
It isn't fixed yet!

**Comment by personalizedrefrigerator** [comment]
Still an issue!

**Comment by personalizedrefrigerator** [comment]
Still an issue!

**Comment by personalizedrefrigerator** [comment]
Still an issue!

**Comment by personalizedrefrigerator** [comment]
Still an issue!

**Comment by namannangia** [comment]
Is someone working on this issue ?

**Comment by roman-r-m** [comment]
> Is someone working on this issue ?

I don't think so

**Comment by gtlsgamr** [comment]
@personalizedrefrigerator I think I fixed it! #7515
